### PR TITLE
Attempt to fix broken coverage reports

### LINKED ISSUE #747: Travis does not report correct coverage to Codecov
Mapknitter is at ~70% test coverage, but due to running tests in parallel, partial test coverage is being sent to codecov.
So it reports only 40% coverage, which only covers the models.

### PR DESCRIPTION
**Pull request by kaustubh-nair**
Fixes #747 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by alaxalves** [comment]
Did it work ? How do I see the generated output?

**Comment by kaustubh-nair** [comment]
Hurray! :tada:

**Comment by kaustubh-nair** [comment]
@alaxalves @jywarren This is ready!

File: .simplecov [hunk 1]
```diff
@@ -1,5 +1,8 @@
+SimpleCov.start
+
 if ENV['CI'] == 'true'
-  SimpleCov.formatter = SimpleCov::Formatter::CoberturaFormatter
```
Comment (alaxalves) [inline .simplecov]: Nice one! This was producing a weird output.

**Comment by cesswairimu** [comment]
Wow awesome :tada: :tada: thanks @kaustubh-nair

**Comment by kaustubh-nair** [comment]
Weird
@alaxalves Any idea why this is failing?

**Comment by alaxalves** [comment]
> Weird
> @alaxalves Any idea why this is failing?

Yes, the dependencies you've added recently might have a version of Bundler > 2.0 requirement, if `gem install bundler` and then `bundle install` with the newly generated lockfile I think that it'll solve the docker build. Or you can manually change the lockfile in [this](https://github.com/publiclab/mapknitter/blob/220c36db60201daa69ea4b0727514cc5e8345108/Gemfile.lock#L343) line to `2.0.1`.
Let me know if it still fails. :D

**Comment by kaustubh-nair** [comment]
@alaxalves Looks like we need to run `gem install bundler` again

**Comment by kaustubh-nair** [comment]
Thanks @alaxalves 
@jywarren This is ready for merge

**Comment by jywarren** [comment]
Super!!! Thanks so much!

**Pull request merged**